controllers/survey.controller.js

```diff
@@ -15,15 +18,86 @@ async function getSurveyData(user_id) {
     });
 }
 
+
+const getdata = (request, response) => {
+    let count = 0;
+    pool.query('SELECT * from survey', (error, results) => {
+        console.log("result--->", results.rows);
+        let data = results.rows
+        data.forEach(item => {
+            console.log("item--->", item);
+            pool.query(
+                "SELECT * from submissions where comment=$1",(comment) => {
+                    item.forEach(item2 => {
```

Comment:
@sonalratho26 
Same as above comment, please update.
Also use valid names in parameter name rather than using `item2`